issue_13/script.js

```diff
@@ -0,0 +1,86 @@
+const modalBtnWrapper = document.getElementById('js-modal-btn__wrapper');
+const modalBtn = document.getElementById('js-modal-btn');
+const modalWrapper = document.getElementById('js-modal__wrapper');
+const modalBack = document.getElementById('js-modal-back');
+const closeBtn = document.getElementById('js-close-btn');
+const requestBtn = document.getElementById('js-request-btn');
+const modalTrigger = [modalBtn, modalBack, closeBtn];
```

Comment:
`js-modal-back`をbuttonにして

```suggestion
const modalButtonElements = [modalBtn, modalBack, closeBtn];
```

Reply:
Thanks for your comments!
I did not have the idea of using the outside of the modal area as a `button` tag.
By doing so, the variable names fit better.
https://github.com/yoshikawa-akr/js-lessons/pull/13/commits/c48a58d9f4d4ca6ff4535fcf8661fb0ed1e25338